Platform/AlderlakeBoardPkg/BoardConfigAdlN.py

```diff
@@ -1,7 +1,7 @@
 ## @file
 # This file is used to provide board specific image information.
 #
-#  Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+#  Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
```

Comment:
Leave copyright year as-is (it is optional now) or change it to 2023 - 2024.

Reply:
updated, thanks for reminding